WordPress/src/main/res/values/strings.xml

```diff
@@ -1481,7 +1481,7 @@
     <string name="error_publish_empty_post">Can\'t publish an empty post</string>
     <string name="error_publish_empty_page">Can\'t publish an empty page</string>
     <string name="error_save_empty_draft">Can\'t save an empty draft</string>
-    <string name="error_publish_no_network">Device is offline. Changes saved locally.</string>
+    <string name="error_publish_no_network">Device is offline. Post saved as a draft.</string>
```

Comment:
I think we should not use  "draft" or "local draft" here.

Reply:
updated the string to "Device is offline. Post Saved locally"